### ISSUE TITLE
[Feature Request] Non-Docker version

### ISSUE THREAD

**Issue by rati0nal** [simojenki/bonob#87]
Is it possible to get a non-docker version of this? The project looks amazing but I don't run a docker environment (LXC install of Navidrome).

**Comment by cooljimy84** [comment]
Just installed Navidrome as subsonic and log4j issues. This as a non docker app would be awesome !

**Comment by simojenki** [comment]
I'll look into this packaging it up as a tgz file so that all is needed is node.js, and some sort of init system to manage the process.

**Comment by jparnzen** [comment]
I've created a dockerless version locally based on the Dockerfile. Just made a very basic script for it as well, and added a systemd service file. Let me know if these would be of use to anyone.

**Comment by simojenki** [comment]
If you provide your script and systemd file I could add them to the README?

**Comment by cooljimy84** [comment]
lxc-create <<name>> -t oci -- --url docker://simojenki/bonob:latest

Or something like that ?

**Comment by jparnzen** [comment]
Sorry for the delay in responding, @simojenki. I’ll try to get the info to you today. What’s the best way to get it to you?

**Comment by simojenki** [comment]
Either add it to this thread, or email simojenki at gmail

**Comment by jparnzen** [comment]
Thanks, @simojenki ! I zipped up the two files (make-bonob-app and bonob.service) and attached it here. Let me know if you have any issues with anything.

Quick notes:
* I run make-bonob-app in the directory above the cloned bonob repo. Accepts one arg--the name of the directory of the clone. Creates a new directory named ${clone}-app with the launchable files. Per the comment at the end of the script, I then copy the *-app dir to /var/lib as the app's new home.
* The systemd service file for bonob is taken from the one for navidrome IIRC. It expects to launch bonob from /var/lib/bonob-app/src (the WorkingDirectory) and needs to have the environment vars for bonob adjusted. It also has a custom env var for flac players (I use it for downsampling hi-res flacs to sonos-appropriate flacs).

Let me know if you have any questions. They're very basic files but hopefully of some use.

[bonob-support-files.zip](https://github.com/simojenki/bonob/files/7848734/bonob-support-files.zip)

**Comment by certuna** [comment]
Yeah another vote for non-Docker, it introduces so many networking issues...

**Comment by simojenki** [comment]
I'll look into using pkg to creating a single executable binary.  This time of year is super busy however so it's going to have to wait a few weeks.

**Comment by LeePorte** [comment]
+1 to this

**Comment by simojenki** [comment]
Perhaps this project could make that happen -> https://github.com/vercel/pkg

**Comment by u8915055** [comment]
Not sure if this still works or whether any further work has been done on this.  I really want to run this as an LXC.  Im working through it but finding the script now fails in a couple places.. I think some of the dependencies are no longer valid.  I will try to post whatever i do to make this work on LXC if i get there.

Currently the error i have is . . . 

```
node-pre-gyp ERR! install response status 404 Not Found on https://github.com/marudor/libxmljs2/releases/download/v0.32.0/node-v120-linux-x64-glibc.tar.gz 
```

Im guessing this is just a versioning issue.. will see if i can modify.  Im just wondering if @jparnzen might have updated this aready.   Thanks so much for the work here!

**Comment by u8915055** [comment]
so when i follow that error above, v120 doesnt seem to be there, the newest is 115 but i have changed reference to 120 in many places but i just cant get it to work.

**Comment by u8915055** [comment]
i could never get this to go.. i assume it needs some updating. i did try another version of this (see other issue here #92 where someone gave the instructions on how to build on Arch Linux.  I created an LXC under proxmox and its working!
